pkg/gui/controllers/local_commits_controller.go

```diff
@@ -430,6 +441,29 @@ func (self *LocalCommitsController) edit(commit *models.Commit) error {
 	})
 }
 
+func (self *LocalCommitsController) quickStartInteractiveRebase() error {
+	if self.c.Git().Status.WorkingTreeState() != enums.REBASE_MODE_NONE {
+		return self.c.ErrorMsg(self.c.Tr.AlreadyRebasing)
+	}
+
+	commit, index, ok := lo.FindIndexOf(self.c.Model().Commits, func(c *models.Commit) bool {
+		return c.IsMerge() || c.Status == models.StatusMerged
+	})
```

Comment:
This should also be a DisabledReason. Maybe not using the FindIndexOf; I guess checking the merge state of `Commits[0]` should be enough for that.

Reply:
Can do